dist/index.html

```diff
@@ -29,12 +28,10 @@ <h2>Recent scores</h2>
       <section id="leaderboard-add">
         <h2>Add your score</h2>
         <form id="add-score">
-          <input type="text" name="name" placeholder="Your name" />
-          <input type="number" name="score" placeholder="Your score" />
           <button type="submit">Submit</button>
         </form>
```

Comment:
- [ ] I think you deleted the inputs for the form but it is needed to add a score, kindly add them again, so you can submit the score